fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -1598,7 +1599,10 @@ public void updateFragmentExecStatus(TReportExecStatusParams params) {
         // and returned_all_results_ is true.
         // (UpdateStatus() initiates cancellation, if it hasn't already been initiated)
         if (!(returnedAllResults && status.isCancelled()) && !status.ok()) {
-            ConnectContext.get().setErrorCode(status.getErrorCode() == null ? "UNKNOWN" : status.getErrorCode().toString());
+            ConnectContext ctx = ConnectContext.get();
```

Comment:
I've a question that, `ConnnectContext` is passed to ctor, could not we just save this context, instead of calling `ConnectContext.get()` all the time?

```
    // Used for new planner
   // <--- ConnectContext != null
    public Coordinator(ConnectContext context, List<PlanFragment> fragments, List<ScanNode> scanNodes,
                       TDescriptorTable descTable) {

    // Used for broker load task/export task coordinator
   // <--- ConnectContext == null
    public Coordinator(Long jobId, TUniqueId queryId, DescriptorTable descTable, List<PlanFragment> fragments,
                       List<ScanNode> scanNodes, String cluster, String timezone, long startTime) {
```

Reply:
Good advice. I will change it.